### PR TITLE
Django 3.1 has been released, include it in the test matrix

### PR DESCRIPTION
**Pull request by ticosax**

### CODE REVIEW

**Comment by ticosax** [comment]
we need to wait for next release of DRF https://github.com/encode/django-rest-framework/commit/e6c1afbcf97e7080c0632ac9e2d60a6d10bd1a5c

**Comment by stianjensen** [comment]
That is released now: https://github.com/encode/django-rest-framework/issues/7459

**Comment by ticosax** [comment]
ready for reviw

File: tests_app/tests/unit/key_constructor/bits/tests.py [hunk 1]
```diff
@@ -471,9 +477,14 @@ def setUp(self):
         self.kwargs['view_instance'].filter_queryset = lambda x: x.filter(is_active=True)
 
     def test_should_use_view__get_queryset__and_filter_it_with__filter_queryset__and_filter_by__lookup_field(self):
-        expected = (u'SELECT "unit_bittestmodel"."id", "unit_bittestmodel"."is_active" '
-                    u'FROM "unit_bittestmodel" '
-                    u'WHERE ("unit_bittestmodel"."is_active" = True {space}AND "unit_bittestmodel"."id" = 123{space})')
+        if django.VERSION >= (3, 1):
+            expected = ('SELECT "unit_bittestmodel"."id", "unit_bittestmodel"."is_active" '
+                        'FROM "unit_bittestmodel" '
+                        'WHERE ("unit_bittestmodel"."is_active" AND "unit_bittestmodel"."id" = 123{space})')
```
Comment (bertonha) [inline tests_app/tests/unit/key_constructor/bits/tests.py:483]: looks like those `space` can be deleted

**Pull request merged**